versioned_docs/version-7.x/static-configuration.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: static-configuration
-title: Getting started with static API
-sidebar_label: Getting started
+title: Overview of static API
+sidebar_label: Overview
 ---
 
 React Navigation primarily uses a dynamic component based API. This provides a lot of flexibility, however there are a few drawbacks:
```

Comment:
This narrative is misleading. As we first said, the static API is recommended, but this one is "primarily used". It's a bit confusing.

Reply:
Gonna delete this page since now fundamentals section covers all of these.